manager/controllers/app/policy_compiler_interface.go

```diff
@@ -83,13 +83,13 @@ func LookupPolicyDecisions(datasetID string, resourceMetadata *datacatalog.Resou
 	openapiResp, err := policyManager.GetPoliciesDecisions(openapiReq, creds)
 	var actions []taxonomy.Action
 	if err != nil {
-		return actions, err
+		return actions, "", err
```

Comment:
in the case of error we don't return openapiResp.Message, but below at Line 108 we do return the message and a new error.
Later in FybricApplicationController, in the case of error, sometimes we add the message into the return error and sometimes do not.

Maybe the logic is correct but please add some comments with explanation.

Reply:
If there is an error from the connector, the response may be nil, invalid, etc.
Added comments with explanation to all relevant functions.